app.js

```diff
@@ -47,8 +47,28 @@ async function create_user(){
 
 async function returnMovie() {
     //this function will return the movie
-    //const api_endpoint = 'rating';
+    const RETURN_MOVIE_URL = 'https://cors-anywhere.herokuapp.com/http://moviefinder.us-east-1.elasticbeanstalk.com/rating/';   
```

Comment:
I think there is value in being consistent with how we handle the endpoint links. Since we use the global variable and CONCAT for the other functions I think it might be good to do the same here. Thoughts? Did you do this for a particular reason?

Reply:
I decided to use a larger constant for the Movie URL in the returnMovie function because the concatenation was giving me some issues in testing and I wanted to make sure that this wasn't an additional step where the method could fail.  Although it is inconsistent, I believe that it would be good to keep it as is for now until we can spend more time on this method in Sprint 2 and become more comfortable with our approach in this method.